ui/components/Policies/Utils/HeaderRows.tsx

```diff
@@ -30,9 +30,10 @@ interface Props {
 const HeaderRows = ({ items }: Props) => {
   return (
     <Flex column gap="8">
-      {items.map((h) => {
-        return (
-          h.visible !== false && (
+      {items
+        .filter((h) => h.visible !== false)
```

Comment:
not all the items contains `visible:true` so that we deal with it as a default value for items to be `visible:true`

for example: 
```
{
    rowkey: "Policy Name",
    value: "Controller ServiceAccount Tokens Automount",
  },
  {
    rowkey: "Application",
    value: "flux-system/external-secrets",
  },
  {
    rowkey: "Violation Time",
    value: "15 minutes ago",
  },
  {
    rowkey: "Category",
    value: " weave.categories.access-control",
    visible: false,
  },
  ```

here we want only to filter over what's not false to save extra checks & make  visible required

Reply:
I see, thanks for the explanation.